hip_data_tools/aws/s3.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+
+from joblib import load
+
+
+class S3Util:
+    def __init__(self, conn, bucket):
+        self.conn = conn
+        self.bucket = bucket
+
+    def _client(self):
```

Comment:
In Java, we define public methods before private methods as a practice. The normal order is public, protected, default, private. We can use the same for python as well.

Reply:
done